console/package.json

```diff
@@ -35,13 +35,13 @@
   ],
   "dependencies": {
     "@emoji-mart/data": "^1.0.8",
-    "@formkit/core": "^1.0.0-beta.12",
-    "@formkit/i18n": "^1.0.0-beta.12",
-    "@formkit/inputs": "^1.0.0-beta.12",
-    "@formkit/themes": "^1.0.0-beta.12",
-    "@formkit/utils": "^1.0.0-beta.12",
-    "@formkit/validation": "1.0.0-beta.12",
-    "@formkit/vue": "^1.0.0-beta.12",
+    "@formkit/core": "^0.16.2",
```

Comment:
Friends who are confused by the version scheme can refer to

![image](https://user-images.githubusercontent.com/16865714/226078871-379786e5-f6bf-4cc5-a974-d72bf1b014ee.png)